xdg-terminal-exec.1

```diff
@@ -0,0 +1,63 @@
+.\"	Title : xdg-terminal-exec
+.\"	Author : Nathan Pratta Teodosio
+.\"	June, 2024
+.\"
+.TH XDG-TERMINAL-EXEC 1 "Xdg-terminal-exec manual"
+.SH NAME
+xdg-terminal-exec \- terminal execution utility
+.SH SYNOPSIS
+\fBxdg-terminal-exec [command [arguments]]
+.BR
+.SH DESCRIPTION
+Xdg-terminal-exec launches the preferred system terminal, as determined
+by the XDG specification.
```

Comment:
I think it would be prudent to mention that the spec is still in proposed state, and this is not an official XDG utility.